.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ commands:
               echo 'Default NVM_DIR to .nvm in home'
               echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
             fi
-            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
+            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" --no-use' >> $BASH_ENV
```

Comment:
NVM tries to pick the node version automatically upon loading the script. For some reason, when using the new CircleCI's docker image, this leads to a failure related to not finding node 14. As a workaround, this behavior has been disabled using the `--no-use` argument. This should have no impact on the job, as we explicitly pick the node version [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/3e05b1577ebe499fc4f738a15c9daff656408bfd/.circleci/config.yml#L154).